Explorer/Assets/DCL/Communities/CommunityDataService/CommunityDataService.cs

```diff
@@ -154,11 +205,14 @@ public bool TryGetCommunity(ChatChannel.ChannelId channelId, out GetUserCommunit
             return communities.TryGetValue(channelId, out communityData);
         }
 
+        public event Action<CommunityMetadataUpdatedEvent>? OnCommunityMetadataUpdated;
```

Comment:
can you move it up before the constructor. Could be lost here in between of functions :)